content/posts/20240311_webhook_site/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Teste de comunicação assíncrona utilizando webhook.site"
+date: "2024-03-11T12:00:00-03:00"
+draft: false
+sidebar: true
+slug: 20240311_webhook_site
+thumbnail:
+  src: "img/thumbnails/20240311_webhook_site.png"
+  visibility:
+    - list
+categories:
+  - "Testes Software"
+tags:
+  - "Testes"
+  - "Tutorial"
+  - "Webhook"
+---
+
+Existem diversas formas de se comunicar assincronamente entre sistemas internos de uma empresa ou até mesmo externos. Hoje em dia muito se fala a respeito de emails (para notificar usuários), menssageria com o [RabbitMQ](https://en.wikipedia.org/wiki/RabbitMQ) e [webhooks](https://en.wikipedia.org/wiki/Webhook). Neste artigo vou explorar sobre o que são os webhooks e também mostrar uma ferramenta que facilita os testes da sua aplicação que dispara webhooks.
+
+<!--more-->
+
+## Introdução
+
+Algumas aplicações precisam possuir recursos para disparar notificações indicando que alguma ação foi executada ou até mesmo notificações via email (normalmente quando é para o usuário). Isso é bem comum em aplicações que possuem processos assíncronos ou demorados, como por exemplo: geração de algum relatório mais complexo, invocação de algum outro sistema assíncrono e até mesmo tarefas mais pesadas.
```

Comment:
Instead of `demorados, como por exemplo:` use `demorados como, por exemplo, geração de`